MAUI/Cartesian-Charts/Axis/Ticklines.md

```diff
@@ -50,6 +50,48 @@ chart.YAxes.Add(secondaryAxis);
 
 {% endtabs %}
 
+## Positioning the ticks
+
+The tick lines can be positioned inside or outside the chart area using the [TickPosition](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_TickPosition) property. [LabelsPosition](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartAxis.html#Syncfusion_Maui_Charts_ChartAxis_LabelsPosition) property default value is `AxisElementPosition.Outside`.
```

Comment:
Update the API as  Tick Position
<img width="747" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/103025761/87b1466e-66e7-4e5c-a292-71ff8bb07706">